pvlib/spectrum/mismatch.py

```diff
@@ -0,0 +1,213 @@
+"""
+The ``mismatch`` module provides functions for spectral mismatch calculations.
+"""
+
+import pvlib
+import numpy as np
+import pandas as pd
+from scipy.interpolate import interp1d
+import os
+
+
+def get_sample_sr(wavelength=None):
+    '''
+    Generate a generic smooth c-Si spectral response for tests and experiments.
+
+    Parameters
+    ----------
+    wavelength: 1-D sequence of numeric, optional
+        Wavelengths at which the spectral response should be interpolated.
+        By default the wavelengths are from 280 to 1200 in 5 nm intervals. [nm]
+
+    Returns
+    -------
+    sr: pandas.Series
+        The relative spectral response indexed by wavelength in nm. [-]
+
+    Notes
+    -----
+    This spectral response is based on measurements taken on a reference cell.
+    The measured data points have been adjusted by PV Performance Labs so that
+    standard cubic spline interpolation produces a curve without oscillations
+    as shown in [1]_.
+
+    References
+    ----------
+    .. [1] Driesse, Anton, and Stein, Joshua. "Global Normal Spectral
+       Irradiance in Albuquerque: a One-Year Open Dataset for PV Research".
+       United States 2020. https://doi.org/10.2172/1814068.
+       https://www.osti.gov/servlets/purl/1814068.
```

Comment:
The links resolve to the same URL, so maybe just keep the doi link. Also, then you could format the URL as a hyperlink.